src/components/organisms/BannerAdmin/BannerAdmin.tsx

```diff
@@ -1,90 +1,176 @@
-import React, { useState, useCallback, useRef } from "react";
+import React, { useCallback, useState, useEffect } from "react";
+import { useForm } from "react-hook-form";
 
 import { makeUpdateBanner } from "api/bannerAdmin";
 
 import { AnyVenue } from "types/venues";
+import { Banner } from "types/banner";
+
+import { useShowHide } from "hooks/useShowHide";
+
+import { Button } from "components/atoms/Button";
+import { Checkbox } from "components/atoms/Checkbox";
+import { InputField } from "components/atoms/InputField";
+
+import { ConfirmationBannerModal } from "./ConfirmationBannerModal";
+
+import "./BannerAdmin.scss";
 
 interface BannerAdminProps {
   venueId?: string;
   venue: AnyVenue;
+  onClose: () => void;
 }
```

Comment:
Is there any reason the `onClose` needs to always be provided? Or can it be made optional?

```suggestion
export interface BannerAdminProps {
  venueId?: string;
  venue: AnyVenue;
  onClose?: () => void;
}
```

Reply:
done 771b70a